### DISCUSSION TITLE
Privileges 2 Public Beta

### DISCUSSION THREAD

**Discussion by mthielemann** [SAP/macOS-enterprise-privileges#106] [Announcements]
Are you excited about the new version of Privileges and would like to test the new functions in advance? Then here is the public beta version of Privileges 2 for you.

Please note that the path to the command line tool has changed and that there have also been changes to the supported options in the configuration profile. Please use the sample profile, which you can find in the app's "Resources" folder, to familiarize yourself with the new options. Privileges 2 requires macOS 11 or newer.

If you find any bugs in the app or in the translation of one of the supported languages, please open an issue in GitHub an add the prefix "[Privileges 2]" to the title so that we can take a look at it and fix the bug. Thank you in advance for your support in testing.

[Privileges_2.0.0b1761.pkg.zip](https://github.com/user-attachments/files/17220657/Privileges_2.0.0b1761.pkg.zip)

**Comment by mthielemann** [comment]
Here's a new build that fixes the reported issues and includes some localization updates.

[Privileges_2.0.0b1775.pkg.zip](https://github.com/user-attachments/files/17277173/Privileges_2.0.0b1775.pkg.zip)

**Comment by mthielemann** [comment]
Here's another new build for the weekend. This fixes a problem that arose with build 1775 and under certain conditions prevented the reason entered by a user from being logged. Have fun testing. 😊

[Privileges_2.0.0b1781.pkg.zip](https://github.com/user-attachments/files/17340723/Privileges_2.0.0b1781.pkg.zip)

**Comment by mthielemann** [comment]
Here's a new build which fixes issue [#115](https://github.com/SAP/macOS-enterprise-privileges/issues/115).

Just a quick reminder... The public beta phase for Privileges 2 ends on October 31st. Thank you to everyone who has tested and provided feedback so far. Thanks to your help we were able to find and fix some minor bugs. Please use the remaining time to continue testing. We look forward to seeing what else you find.

Have fun testing. 😊

[Privileges_2.0.0b1787.pkg.zip](https://github.com/user-attachments/files/17457077/Privileges_2.0.0b1787.pkg.zip)

**Reply by charaschoe** [reply]
Hey @mthielemann, unfortunately there is no file attached to this message. Thanks for your work!

**Reply by mthielemann** [reply]
But now… 😂

**Reply by charaschoe** [reply]
> But now… 😂

Vielen Dank! :)

**Reply by timcheetah** [reply]
Great - was just about to report this issue. Things are looking great so far!

**Comment by mthielemann** [comment]
Here's a new build which fixes issue #117 and adds the functionality requested in issue #116. Happy testing! 😊

[Privileges_2.0.0b1795.pkg.zip](https://github.com/user-attachments/files/17487520/Privileges_2.0.0b1795.pkg.zip)

**Comment by mthielemann** [comment]
Here's a new build that fixes issue #118 and adds a new configuration key `ReasonCheckingEnabled` which (if enabled) should (hopefully) prevent users from specifying reasons like `akdhfgakjsdfgaiuzsdfsjezew`. Happy testing! 😊

[Privileges_2.0.0b1805.pkg.zip](https://github.com/user-attachments/files/17505400/Privileges_2.0.0b1805.pkg.zip)

**Comment by mthielemann** [comment]
Here's a new build for the weekend that

- adds the functionality requested with issues [#120](https://github.com/SAP/macOS-enterprise-privileges/issues/120) and [#123](https://github.com/SAP/macOS-enterprise-privileges/issues/123) 
- introduces a new configuration key `HideSettingsButton` which (if enabled) hides the app's `Settings…` button
- introduces a new configuration key `HideSettingsFromDockMenu` wich (if enabled) hides the new `Settings…` entry of the Dock tile menu

Happy testing! 😊

[Privileges_2.0.0b1815.pkg.zip](https://github.com/user-attachments/files/17518058/Privileges_2.0.0b1815.pkg.zip)

**Comment by mthielemann** [comment]
The public beta phase of Privileges 2 has now ended. Many thanks to everyone who has tested and gave us feedback over the last 4 weeks, so we were able to fix some minor bugs and even implement some small improvements. Thank you very much! As announced, we will be releasing Privileges 2 to the public in November, following our final internal testing and QA.
